### PR TITLE
Implementing equality operator for AzColor and Color objects.

### PR DESCRIPTION
**Pull request by Adenilson**
This should easy the use of AzColor by servo (e.g. layout_task.rs).

### CODE REVIEW

File: src/azure.rs [hunk 1]
```diff
@@ -253,6 +253,16 @@ pub struct struct__AzColor {
 
 pub type AzColor = struct__AzColor;
 
+#[inline]
+impl PartialEq for AzColor {
+    fn eq(&self, other: &AzColor) -> bool {
+        if self.r != other.r || self.g != other.g || self.b != other.b || self.a != other.a {
```
Comment (mrobinson) [inline src/azure.rs]: Why not just do:

self.r == other.r && self.g == other.g...

and skip the conditional block?

File: src/azure_hl.rs [hunk 2]
```diff
@@ -125,6 +125,16 @@ impl Color {
     }
 }
 
+#[inline]
+impl PartialEq for Color {
```
Comment (mrobinson) [inline src/azure_hl.rs]: Same comment here. :)

File: src/azure.rs (hunk 1 above)
Comment (Adenilson) [inline src/azure.rs]: Yeah, it makes sense and the code will be cleaner. Let me update it.

File: src/azure_hl.rs (hunk 2 above)
Comment (Adenilson) [inline src/azure_hl.rs]: Ok, done.

**Comment by mrobinson** [comment]
Looks good to me!

**Pull request merged**